vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,321 @@
+---
+title: "Tutorial: Install a dependency from a manifest file"
+description: Learn to install your projects dependencies with vcpkg by using a manifest file.
+author: vicroms
+ms.author: viromer
+ms.date: 8/10/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+---
+# Tutorial: Install a dependency from a manifest file
+
+vcpkg has two operation modes: classic mode, and manifest mode. This article describes how to install packages using manifest mode, which is the recommended workflow for most users.
+
+In manifest mode, you declare your project's direct dependencies in a manifest file named
+[`vcpkg.json`](../reference/vcpkg-json.md).
+
+Manifest files have their own `vcpkg_installed` directory where they install their dependencies,
+unlike classic mode, where all packages are installed in a common `%VCPKG_ROOT%/installed`
+directory. Manifests are suitable for using vcpkg with multiple codebases, as each codebase can have
+a different set of dependencies that do not interfere with each other.
```

Comment:
Should we avoid making comparisons between manifest mode and classic mode? I think we should just mention manifest-related topics. If we decide to use a zone group to differentiate between the both we can just !include the manifest article.